week-2/E-conditionals/exercise.js

```diff
@@ -11,10 +11,11 @@ var role = "mentor";
 var name2 = "Harun";
 var role2 = "student";
 
-
+if (role=="student"){
```

Comment:
When comparing strings, you should use three equals signs: `role === "student"`.